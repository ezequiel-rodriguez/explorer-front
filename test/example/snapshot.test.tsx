import { render } from '@testing-library/react'
import TestPage from './TestPage'
 
describe('Testing Example - Snapshot', () => {
  it('renders homepage unchanged', () => {
    const { container } = render(<TestPage />)
    expect(container).toMatchSnapshot()
  })
})