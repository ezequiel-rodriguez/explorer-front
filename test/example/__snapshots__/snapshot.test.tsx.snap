// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Example - Snapshot renders homepage unchanged 1`] = `
<div>
  <div>
    <h1>
      Home
    </h1>
    <a
      href="/about"
    >
      About
    </a>
  </div>
</div>
`;
